tensorflow_addons/optimizers/novograd_test.py

```diff
@@ -0,0 +1,140 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tests for Novograd Optimizer."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow as tf
+import numpy as np
+
+from tensorflow_addons.utils import test_utils
+from tensorflow_addons.optimizers import Novograd
+
+
+@test_utils.run_all_in_graph_and_eager_modes
+class NovogradTest(tf.test.TestCase):
```

Comment:
Could we also add a test method for `grad_averaging=True`?

Reply:
Done.